desktop-environment.el

```diff
@@ -250,6 +298,23 @@ portion of the screen."
            (shell-command-to-string desktop-environment-volume-toggle-microphone-command)))
 
 
+;;; Commands - keyboard backlight
+;;;###autoload
+(defun desktop-environment-keyboard-backlight-increment ()
+  "Increment keyboard backlight by `desktop-environment-keyboard-normal-increment'."
```

Comment:
the variable `desktop-environment-keyboard-normal-increment` does not exist. Its name is `desktop-environment-keyboard-backlight-normal-increment`.

Reply:
Fixed.